### DISCUSSION TITLE
Failed to initiate the GitHub login process. Please try again.

### DISCUSSION THREAD

**Discussion by JanisBe** [community/community#17295] [Copilot Conversations]
Description: 
Copilot stopped working after some time. Can't login, whenever I'm trying to log, I get "Failed to initiate the GitHub login process. Please try again."
Log:
`2022-05-26 09:04:03,279 [ 245451]   WARN - #c.g.c.l.a.CopilotAgentProcessServiceImpl - Failed to execute JSON-RPC command or notification. Request: 6, command: signInInitiate
java.io.IOException: Stream Closed
	at java.base/java.io.FileOutputStream.writeBytes(Native Method)
	at java.base/java.io.FileOutputStream.write(FileOutputStream.java:354)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
	at java.base/java.io.BufferedOutputStream.write(BufferedOutputStream.java:127)
	at java.base/java.io.FilterOutputStream.write(FilterOutputStream.java:108)
	at com.github.copilot.lang.agent.vscodeRpc.VSCodeJsonRpc.sendBytes(VSCodeJsonRpc.java:77)
	at com.github.copilot.lang.agent.vscodeRpc.VSCodeJsonRpc.sendCommand(VSCodeJsonRpc.java:39)
	at com.github.copilot.lang.agent.CopilotAgentProcessServiceImpl.lambda$executeCommandWithPromise$3(CopilotAgentProcessServiceImpl.java:274)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)`

I use IntelliJ IDEA 2022.1.1 (Ultimate Edition) Build #IU-221.5591.52, built on May 10, 2022, Copilot version 1.1.23.1570

**Comment by akheron** [comment]
I have exactly the same problem! I just got access to Copilot Preview after a long wait, and now I can't use it :(

**Comment by badandboy** [comment]
+1

**Comment by midFang** [comment]
use dnsjumper app to flush dns and set dns 8.8.8.8, then it works.

**Reply by badandboy** [reply]
it doesn`t work for me....

**Reply by freesme** [reply]
After refreshing DNS try restart IDEA and then log in to Copilot

> it doesn`t work for me....

**Reply by JanisBe** [reply]
didn't work for me either...

**Comment by freesme** [comment]
have you tried to reinstall the latest version copilot？

**Reply by JanisBe** [reply]
couple of times, I even reinstalled IJ, still getting the same error :/

**Reply by freesme** [reply]
> 
dnsjumper:  https://drive.google.com/file/d/1cCkVaEvg8rDqIfeqdqFA2CQ1g_dwQAbL/view?usp=sharing
![step](https://user-images.githubusercontent.com/56824280/170681387-66954137-0a82-4537-bba2-e14a301b651d.png)

if you don't logout, click <logout from Github> first
![image](https://user-images.githubusercontent.com/56824280/170681142-3ad75ba8-b930-44a3-a74b-f0e8ca7a6edc.png)

Hope to useful

**Reply by jhao14** [reply]
it works for me

**Reply by LarissaLycoris** [reply]
有用我解决了

**Reply by didi12121** [reply]
ORZ大佬

**Reply by freesme** [reply]
PS: The use of refreshing DNS may lead to slower access to domestic websites in China, which can be modified to the domestic DNS after resolving Copilot's issues

**Reply by szf-2000** [reply]
i am macOS, how can i  do this.

**Reply by RanY-Luck** [reply]
这个对我有用，但是我mac os 怎么解决呢

**Reply by didi12121** [reply]
> 这个对我有用，但是我mac os 怎么解决呢

装windows (狗头)

**Reply by freesme** [reply]
> 这个对我有用，但是我mac os 怎么解决呢

1.刷新DNS网上大把方法  2.或者多改几次代理地址试试

**Reply by szf-2000** [reply]
restart idea

**Reply by Chronicle-495** [reply]
nb！

**Reply by PengKun1** [reply]
it works for me

**Comment by jansorg** [comment]
This may be related to a proxy server. Please refer to https://github.com/github/feedback/discussions/16230#discussioncomment-2747034

**Comment by yuchdev** [comment]
Is someone going to do something about this issue?
I DO NOT use any proxy servers, and still I get the same error.
We pay for this service, and it does not work at all.

**Comment by orhnk** [comment]
> [!IMPORTANT]
> Only if you are using NixOS
> https://nixos.wiki/wiki/Jetbrains_Tools

**Reply by xbz-24** [reply]
![image](https://github.com/community/community/assets/68678258/6f1718ea-5b6e-4947-96ba-58cf70062e4f)

thx, this did the trick for me!!!
you are awesome

**Comment by Vishal008-Vichu** [comment]
Has anyone here found a solution for this? I’m encountering the same error, yet everything appears to be set up correctly. Despite raising this in multiple forums, the Copilot team hasn't responded to any tickets, which is disappointing given that it’s a paid service.
